buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -40,10 +40,10 @@
  *              win_x86-64_cmprssptrs,
  *              win_x86 (Java 8 support only),
  *              zos_390-64_cmprssptrs (Java 11 support only),
- *              osx_x86-64 (Java 8 and Java 11 support only),
- *              osx_x86-64_cmprssptrs (Java 8 and Java 11 support only),
- *              aarch64_linux (Java 8 and Java 11 support only),
- *              aarch64_linux_xl (Java 8 and Java 11 support only)
+ *              osx_x86-64,
+ *              osx_x86-64_cmprssptrs,
```

Comment:
See https://github.com/eclipse/openj9/pull/10614#discussion_r489604215.

Reply:
See if you like the latest change, and then I'll change the other PR to match.

Reply:
Updated again.

Reply:
That works for me.

Reply:
Made another change, pls review again.

Reply:
@AdamBrousseau I'm still not sure if this is correct, can you pls review.

Reply:
@keithc-ca Adam isn't available to review in the short term. Are you ok to merge as-is, and we can make any further corrections later? I'd like to run a jdk15 aarch64 build which is blocked by getting this merged, and this has already been delayed from yesterday.